browser/brave_shields/brave_shields_tab_helper.cc

```diff
@@ -159,6 +159,13 @@ std::vector<GURL> BraveShieldsTabHelper::GetFingerprintsList() {
   return fingerprints_list;
 }
 
+std::vector<ContentSettingsType>
+BraveShieldsTabHelper::GetInvokedWebcompatFeaturesList() {
+  std::vector<ContentSettingsType> features_list(
+      webcompat_features_invoked_.begin(), webcompat_features_invoked_.end());
+  return features_list;
```

Comment:
consider returning the map by const ref and converting it to vector only where required.

Reply:
done